app/assets/stylesheets/govuk_publishing_components/components/_task-list.scss

```diff
@@ -413,14 +394,33 @@ $top-border: solid 2px $grey-3;
       }
     }
   }
+
+  .gem-c-task-list__link-item {
+    color: $black;
+
+    &:visited,
+    &:active,
+    &:link {
+      color: $black;
+    }
+  }
+}
+
+.gem-c-task-list__link-item {
+  text-decoration: none;
+
+  &:hover {
+    text-decoration: underline;
+  }
 }
 
 .gem-c-task-list__context {
+  display: inline-block;
```

Comment:
It's needed to keep the dash next to it when it's wrapping at the end of a line. Otherwise you can end up with a situation that looks like this:
```
The text of a really long link that goes almost right across -
£25
```